src/WPML_OptionsManager.php

```diff
@@ -537,7 +537,27 @@ public function _LogMenu() {
                 $emailLogList->display();
                 ?>
             </form>
-            <?php $this->displayMP3Banner() ?>
+            <?php
+            $banner = false;
+            ob_start();
+            $this->displayMP3Banner();
+            $banner = ob_get_contents();            
+            ob_end_clean();
+            /**
+            * Control whether the banner is displayed or not
+            *
+            * @since 1.9.7
+            *
+            * @param boolean $display Whether the banner should be displayed. (default: true)
+            * @param string  $banner  The default banner content
+            * 
+            * @return boolean
+            */
+            $display = apply_filters('wpml_banner_display', true, $banner );
+            if ( $display ) {
+                echo $banner;
```

Comment:
Could you please eliminate the output buffering, replacing the `echo $banner;` with the `$this->displayMP3Banner();` call, as you've suggested in your initial PR?
Now that you have eliminated the banner customization, this would be a nice simplification.
Once done, I think we can merge the PR.